app/Filament/Resources/ResultResource.php

```diff
@@ -311,6 +314,9 @@ public static function table(Table $table): Table
                     ->badge()
                     ->toggleable()
                     ->sortable(),
+                Tables\Columns\TextColumn::make('comments')
```

Comment:
Can you remove comments from the table, I'm looking to keep the table as clean as possible with just metrics.